File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int n = nums.size(), i=0;
        for(int x=0;x<n;x++){
            if(nums[x] != val){
                nums[i++] = nums[x];
            }
        }
        return i;
    }
};

File: Data Structures & Algorithms/rotate-array/submission-0.java
class Solution {
    public void rotate(int[] nums, int k) {
        int n = nums.length;
        k %= n;
        int[] arr = new int[n];
        int x = 0;
        for(int i=n-k;i<n;i++){
            arr[x++] = nums[i];
        }
        for(int i=0;i<n-k;i++){
            arr[x++] = nums[i];
        }
        for(int i=0;i<n;i++){
            nums[i] = arr[i];
        }
    }
}